client-course-schedulizer/src/components/Header/ImportButton/ImportButton.scss

```diff
@@ -5,3 +5,11 @@
 .hidden {
   display: none !important;
 }
+
+.import-label {
+  padding-top: 6px !important;
+  padding-bottom: 6px !important;
+  padding-left: 16px !important;
+  padding-right: 16px !important;
+  cursor: pointer !important;
+}
```

Comment:
Since these are the same styles as the export button, I would combine both of the styles and put the styles in an `.scss` file for the parent component (I think header.scss or something similar). Then you could have only one .hidden class and then `.import-label, .export-label { ... }`.

Reply:
7a9bda1